v2/client/src/actions/users.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+import * as types from '../constants/actionTypes';
+
+export const fetchTrainerResults = () => async dispatch => {
```

Comment:
unless for a very good reason, if this is the only thing to export, better use export default

be consistent with other export